src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -5,92 +5,92 @@
     private static final float LOAD_FACTOR = 0.75f;
     private Node<K,V>[] table;
     private int capacity = INITIAL_CAPACITY;
-    private int size = 0;
-    private int threshold = (int) (capacity * LOAD_FACTOR);
+    private int size;
+    private int threshold;
 
     MyHashMap() {
         table = (Node<K, V>[]) new Node[capacity];
+        size = 0;
+        threshold = (int) (capacity * LOAD_FACTOR);
     }
 
-    private int getHash(Object key, int capacity) {
+    private int getBucketIndex(Object key, int capacity) {
         return key == null ? 0 : key.hashCode() & (capacity - 1);
```

Comment:
Create 2 methods: 
- hachCode(K key) without using capacity at all; This hach (not bucket) you should put in hash field of node;
- getBrucketIndex(int hash) for get returning hash % capacity